fix_frontmatter.py

```
@@ -0,0 +1,103 @@
+                line = body_lines[-i].strip()
+                if line == '---' and i > 1:
+                    is_junk = True
+                    break
+                if ':' in line:
+                    key = line.split(':', 1)[0].strip()
+                    if key in [f.replace(':','') for f in junk_fields]:
+                        val = line.split(':', 1)[1].strip()
+                        if val:
+                            try:
+                                junk_data[key] = yaml.safe_load(val)
+                            except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)